explore-analyze/dashboards.md

```diff
@@ -9,14 +10,42 @@ products:
   - id: kibana
 ---
 
-# {{kib}} dashboards [dashboard]
+# Dashboards [dashboard]
 
-Dashboards are the best way to visualize and share insights from your {{es}} data.
+**Dashboards** provide the primary way to visualize and share insights from your {{es}} data. They allow you to build interactive displays that combine multiple visualizations, metrics, and controls into a single view that helps you and your team monitor trends, investigate issues, and make data-driven decisions.
 
-A **dashboard** is made of one or more **panels** that you can organize as you like. Each panel can display various types of content: **visualizations** such as charts, tables, metrics, and maps, **static annotations** like text or images, or even **specialized views** for Machine Learning or Observability data.
+A dashboard is composed of one or more **panels** that you arrange to tell your data story. Each panel can display visualizations such as charts, tables, metrics, and maps, static annotations like text or images, or specialized views for {{ml}} or {{observability}} data.
 
 ![Example dashboard](/explore-analyze/images/kibana-dashboard-overview.png "")
 
-At any time, you can [share a dashboard](dashboards/sharing.md) you’ve created with your team, in {{kib}} or outside.
+## Get started with dashboards [get-started]
 
-Some dashboards are created and managed by the system, and are identified as `managed` in your list of dashboards. This generally happens when you set up an integration to add data. You can’t edit managed dashboards directly, but you can [duplicate](dashboards/duplicate-dashboards.md) them and edit these duplicates.
+New to dashboards? Start with these hands-on tutorials that walk you through creating your first dashboards using sample data:
+
+- [Create a simple dashboard to monitor website logs](dashboards/create-dashboard-of-panels-with-web-server-data.md): Build a dashboard with charts, metrics, and filters to analyze web traffic patterns
```

Comment:
You might need to add a period when the list item contains a complete sentence.
RE https://stunning-adventure-qrvr1k2.pages.github.io/style-guide/formatting/#lists